content/news/core-monthly/2023/core-monthly-2023-01-01-2023-01-31.md

```diff
@@ -0,0 +1,617 @@
+
+![Core Monthly banner](/assets/images/theme/banner-core-monthly-wide.jpg)
+
+Yes, that really was a very busy January! The current dynamic in the project is at the top level. What happened in January?
+
+### PrestaShop 8.0.1
+
+PrestaShop 8.0.1 was released on January 6th, and it is the first patch version for PrestaShop 8.0. This release includes various bug fixes and improvements, you can read more [in the release notes]({{< relref "/news/releases/prestashop-8-0-1-maintenance-release" >}}).
+
+### Github Discussions are now open!
+
+GitHub Discussions are now [enabled for the PrestaShop Project](https://github.com/PrestaShop/PrestaShop/discussions) 🎉 If you don't feel like using Slack, want to discuss your ideas conveniently or have questions, that's the excellent place to start!
+You can [read a full announcement here]({{< relref "/news/posts/2023/github-discussions-are-now-enabled-in-the-prestashop-project" >}}).
+
+### Feature freeze for PrestaShop 8.1.0
+
+At the end of January, PrestaShop 8.1 [has entered feature freeze]({{< relref "news/releases/prestashop-8-1-feature-freeze" >}}), meaning no new features will be added, and the focus is now on polishing existing features before release. A public beta is expected to be released in a couple of weeks, and the community is encouraged to participate in testing and provide feedback. The most notable features in PrestaShop 8.1 include improved product pages, image generation in additional formats, better control over discontinued products, and more. I encourage you to [subscribe to the GitHub issue](https://github.com/PrestaShop/PrestaShop/issues/30719), so you can stay updated on the release of 8.1.
+
+That also means that work on PrestaShop 9.0 has begun! Contributions introducing backward-incompatible changes can now be merged into the `develop` branch for the next major version. If you have any ideas and want to contribute [we can discuss them](https://github.com/PrestaShop/PrestaShop/discussions)! Make sure to follow project communication channels for more updates on PrestaShop 9.0 soon.
+
+### Module releases
+
+As you can see later in the article, we released many new versions of the modules in January. It wouldn't be possible without a huge work from [Daniel Hlavacek (@Hlavtox)](https://github.com/Hlavtox) and our amazing Quality Assurance team! You can see a release of a `ps_viewedproduct` module, with a [fix for a popular bug with PrestaShop cookie](https://github.com/PrestaShop/ps_viewedproduct/pull/25) that was reported many times. There's also a [new version](https://github.com/PrestaShop/ps_emailsubscription/releases/tag/v2.7.1) of `ps_emailsubscription` with some [performance optimalizations](https://github.com/PrestaShop/ps_emailsubscription/pull/89) and an improvement, so that developres can [perform additional actions after customer's subscription during registration](https://github.com/PrestaShop/ps_emailsubscription/pull/84/files).
```

Comment:
```suggestion
As you can see later in the article, we released many new versions of the modules in January. It wouldn't be possible without a huge work from [Daniel Hlavacek (@Hlavtox)](https://github.com/Hlavtox) and our amazing Quality Assurance team! You can see a release of a `ps_viewedproduct` module, with a [fix for a popular bug with the PrestaShop cookie](https://github.com/PrestaShop/ps_viewedproduct/pull/25) that was reported many times. There's also a [new version](https://github.com/PrestaShop/ps_emailsubscription/releases/tag/v2.7.1) of `ps_emailsubscription` with some [performance optimalizations](https://github.com/PrestaShop/ps_emailsubscription/pull/89) and an improvement, so that developers can [perform additional actions after the customer's subscription during registration](https://github.com/PrestaShop/ps_emailsubscription/pull/84/files).
```